src/components/Footer/Footer.js

```diff
@@ -10,8 +10,16 @@ import MemoIc_block_inactive from '../../assets/block/Ic_block_inactive';
 import MemoIc_comment from '../../assets/icons/Ic_comment';
 import MemoIc_share from '../../assets/icons/Ic_share';
 import Memoic_globe from '../../assets/icons/ic_globe';
-import {COLORS, FONTS} from '../../utils/theme';
+import MemoIc_senddm from '../../assets/icons/ic_send_dm';
+import SendDMAnonBlock from '../../assets/icons/images/send-dm-anon-black.svg';
+import SendDMBlack from '../../assets/icons/images/send-dm-black.svg';
+import {Context} from '../../context';
+import useDMMessage from '../../hooks/core/chat/useDMMessage';
+import useCreateChat from '../../hooks/screen/useCreateChat';
 import BlurredLayer from '../../screens/FeedScreen/elements/BlurredLayer';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
+import {COLORS, FONTS} from '../../utils/theme';
+import BottomSheetMenu from '../BottomSheet/BottomSheetMenu';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New imports have been added to support the direct messaging feature. These include new icons, hooks for managing chat and direct messages, and a BottomSheetMenu component. No issues found here.
```diff
+ import MemoIc_senddm from '../../assets/icons/ic_send_dm';
+ import SendDMAnonBlock from '../../assets/icons/images/send-dm-anon-black.svg';
+ import SendDMBlack from '../../assets/icons/images/send-dm-black.svg';
+ import {Context} from '../../context';
+ import useDMMessage from '../../hooks/core/chat/useDMMessage';
+ import useCreateChat from '../../hooks/screen/useCreateChat';
+ import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
+ import BottomSheetMenu from '../BottomSheet/BottomSheetMenu';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->